### DISCUSSION TITLE
LiaScript LiveEditor Export to GitHub gist

### DISCUSSION THREAD

**Discussion by JMueller-edu** [LiaScript/LiaScript#149] [Q&A] [answered]
Hallo liebe LiaScript Community,

funktioniert bei euch im **LiaScript LiveEditor** der **Export to GitHub gist**?
Bei mir öffnet sich immer nur eine neue Seite mit der Meldung **Exporting**.
Muss ich vorher irgendwo einen Bezug zu meinem GitHub Account herstellen?
**Was mache ich falsch?**

Mit besten Grüßen
Jan

**Comment by andre-dietrich** [comment, marked as answer]
Hallo Jan,

nein du musst nichts ändern, da ist ein kleiner Bug, wenn du nur eine Datei ohne eigene Bilder exportierst, dann sollte es funktionieren. Den Upload von im Browser hochgeladenen Dateien und Multimedia haben wir noch nicht implementiert ... sind aber dran ;-)

Wenn es bei nur Text nicht funktioniert, kannst du dann bitte mal ein Screenshot machen und sagen, welchen Browser du verwendest? 

Liebe Grüße
André

**Reply by JMueller-edu** [reply]
Hallo André,

vielen lieben Dank für die schnelle Antwort.
Ich habe es nochmal nach deinem Vorgehen getestet und es versucht mit Screenshots zu dokumentieren.

![image](https://github.com/LiaScript/LiaScript/assets/105539256/99ccdcec-f0f3-43c6-9687-310a28c0dde3)
![image](https://github.com/LiaScript/LiaScript/assets/105539256/dbc9c46a-9287-4052-8a18-d9bc3043fed6)
![image](https://github.com/LiaScript/LiaScript/assets/105539256/af2007e9-fb3b-481f-b4bd-aef5ec5b886f)

leider hat es im Edge-Browser nicht funktioniert.
Im Brave-Browser dagegen konnte ich den gist export erfolgreich durchführen.

Mit besten Grüßen
Jan
